src/reference/asciidoc/kafka.adoc

```diff
@@ -149,6 +149,55 @@ You may wish to invoke `flush()` before waiting or, for convenience, the templat
 parameter which will cause the template to `flush()` on each send.
 Note, however that flushing will likely significantly reduce performance.
 
+[[transactions]]
+===== Transactions
+
+The 0.11.0.0 client library added support for transactions.
+Spring for Apache Kafka adds support in several ways.
+
+- `KafkaTransactionManager` - used with normal Spring transaction support (`@Transactional`, `TransactionTemplate` etc).
```

Comment:
is the more "fancy" transactional stuff like PROPAGATION_REQUIRES_NEW and PROPAGATION_NESTED also supported ?